arch/arm/boot/dts/nintendo3ds.dtsi

```diff
@@ -157,63 +157,78 @@
 			interrupts = <GIC_SPI 0x35 IRQ_TYPE_EDGE_RISING>;
 
 			mcu: mcu@25 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+
 				compatible = "nintendo,3ds-mcu";
 				reg = <0x25>;
 
 				mcuintc: mcu-intc {
 					compatible = "nintendo,3dsmcu-intc";
 
+					reg = <0x10>;
+
 					interrupt-controller;
 					#address-cells = <1>;
 					#interrupt-cells = <2>;
 
 					interrupts-extended = <&gpio3 9 IRQ_TYPE_EDGE_FALLING>;
 				};
 
-				led: mcu-led {
+				mcu-led {
 					compatible = "nintendo,3dsmcu-led";
+					reg = <0x2D>;
 				};
 
-				rtc: mcu-rtc {
+				mcu-rtc {
 					compatible = "nintendo,3dsmcu-rtc";
+					reg = <0x30>;
 				};
 
-				powersupply: mcu-powersupply {
-					compatible = "nintendo,3dsmcu-powersupply";
+				mcu-charger {
+					compatible = "nintendo,3dsmcu-charger";
+					reg = <0xA>;
 				};
 
-				accelerometer: mcu-accel {
+				mcu-accel {
 					compatible = "nintendo,3dsmcu-accel";
+					reg = <0x40>;
 				};
 
-				pwroff_reg: pwroff-regulator {
+				pwroff_reg: poweroff-regulator@20 {
```

Comment:
if the convention is to have `@<reg>` suffixes, should the above 4 `mcu-` nodes follow that convention as well?

Reply:
Yup, forgot to add those, will fix